### PR TITLE
confirmation-page: create transaction

### PR DESCRIPTION
**Pull request by vcapretz**
## Description

the shipping default state didn't correspond to the selected value, now
it does!

the value type on dropdown component was just string, we need it to be
both string or number

## Your checklist for this pull request

:rotating_light: Please review these items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe with this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

**Comment by vcapretz** [comment]
hey @augusto-jm-amaral I just opened this PR, I need your help making sure the final transaction info (selected payment, amount, etc) comes to the Confirmation page so we can send it to the API and make the transaction 👍

File: webapp/src/containers/Checkout/index.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ class Checkout extends Component {
       closingEffect: false,
       isBigScreen: true,
       footerButtonVisible: true,
+      addresses: [],
```
Comment (thiagommedeiros) [inline webapp/src/containers/Checkout/index.js]: nao seria melhor `shippingAddresses` já que somente a ShippingPage vai lidar com esse array?

**Comment by GitCop** [comment]
Os seguintes commits de seu Pull Request precisam de ajustes no título:

 * Commit: dac1f12438f56198734b1d6816d9c9813d24c92b
  * Utilize o padrão `escopo: ação descrição`, sempre em minúsculo e inglês.  
  
 - O **escopo** não deve conter espaços. Se a palavra for composta, separe por traços  
    - A **ação** deve ser **add**, **remove**, **rename**, **refactor**, **move**,  **bump**, **update** ou **release**, **fix**  
    - A **descrição** deve conter uma rápida descrição da alteração

O guia completo de estilo de commits está disponível em https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by augusto-jm-amaral** [comment]
Moved to https://github.com/pagarme/artis/pull/129

**Pull request closed without merging**